src/CSV.jl

```diff
@@ -52,6 +76,8 @@ getnames(f::File) = getfield(f, :names)
 gettypes(f::File) = getfield(f, :types)
 getrows(f::File) = getfield(f, :rows)
```

Comment:
I'm wondering if `getnrows` and `getncols` is better (the distinction between `getcols` and `getcolumns` is confusing). Similarly the `File` struct could have field names `nrows` and `ncols`.

Reply:
These aren't meant to be publicly used at all, so I didn't put a ton of thought into the names.